components/dashboard/src/workspaces/BrowserExtensionBanner.tsx

```diff
@@ -66,10 +67,12 @@ const getDeduplicatedScmProviders = (user: User, descriptions: AuthProviderDescr
         .filter(isIdentity)
         .map((provider) => provider.type);
 
-    return userProviders
+    const unifiedProviders = userProviders
         .map((type) => unifyProviderType(type))
         .filter(isAuthProviderType)
         .sort();
+
+    return uniq(unifiedProviders);
```

Comment:
here's the actual change. Opted for using the uniq function from lodash for better readability, compared to `[...new Set(x)]`. Performance should not be a deciding factor here either way.

Reply:
How about add lodash/uniq only? Not sure if we have tree shake for dashboard now

Reply:
@mustard-mh I followed [this doc](https://lodash.com/per-method-packages) from their documentation:
> Lodash methods are available in standalone [per method packages](https://www.npmjs.com/browse/keyword/lodash-modularized) like lodash.mapvalues, lodash.pickby, etc. These packages contain only the code the method depends on.
However, use of these packages is discouraged and they [will be removed in v5](https://github.com/lodash/lodash/issues/3838#issuecomment-398592530).

Reply:
I see, then we need to update BrowserExtensionBanner.tsx to use `import uniq from "lodash/uniq"` as with this way, it can remove unnecessary functions. 

(I double checked according to [this article](https://tjaddison.com/blog/2020/12/analyze-your-create-react-app-bundle-size-without-ejecting/) current code will bundle whole lodash, after modify the big lodash block is gone)


> But lodash isn't lightweight enough!
> Don't worry—if you import or require methods directly, e.g. const throttle = require('lodash/throttle'), only the subset of lodash code your package uses will be bundled in projects that use your package.

Reply:
Thank you for sharing the doc @filiptronicek !! I just know it. TIL

Reply:
|From| To|
|:-:|:-:|
|![SCR-20240429-nbvu](https://github.com/gitpod-io/gitpod/assets/20944364/8042dd0e-6721-4e54-9b1b-1e2d7d16f0f2)|![SCR-20240429-nbqg](https://github.com/gitpod-io/gitpod/assets/20944364/408d96f5-0aca-467e-b504-b9ec8c035649)|

Reply:
sounds good, updated the code. thanks for your investigation!